### PR TITLE
Add wooden roller coaster trains and single person swinging cars

### PR DESCRIPTION
**Pull request by spacek531**
These rides have a lot of sprites - similar to steel coasters in the previous RCT1 rides pack.

Deviating from the beautified standard for the sprites list can deflate file sizes - the reversed wooden coaster object file shrinks from 47KB to 33KB just by removing the eight spaces preceding the image list entries from all entries. Is this desirable?

### CODE REVIEW

**Comment by ZeeMajora** [comment]
You seem to have accidentally used outdated versions of the wooden coaster trains, they're missing the name changes & proper reversed train rating multipliers from the last PR i did for them in our repo.

File: objects/rct1/ride/rct1.ride.wooden_rc_trains_reversed/object.json [hunk 1]
```diff
@@ -0,0 +1,1454 @@
...
+    "objectType": "ride",
+    "properties": {
+        "type": "classic_wooden_rc",
+        "category": "rollercoaster",
+        "minCarsPerTrain": 2,
+        "maxCarsPerTrain": 7,
+        "ratingMultipler": {
+            "excitement": 2,
+            "intensity": 10,
+            "nausea": 15
```
Comment (SpartanFrederic104) [inline objects/rct1/ride/rct1.ride.wooden_rc_trains_reversed/object.json:16]: "intensity": **12**, 
"nausea": **22**

**Review by ZeeMajora — APPROVED** [review]

**Comment by Gymnasiast** [comment]
I have pulled this, but I’m not 100% sure about the ID or the name (specifically, I’m considering whether we should add “Classic”). I might still modify that before it hits the `develop` branch of OpenRCT2 proper.

**Comment by ZeeMajora** [comment]
We didn't add "Classic" to the steel mini trains' names, and we did initially use classic for the wooden coaster trains (they were formerly "Classic Wooden Coaster Trains") but we decided to rename it to remove classic from the name for consistency with the steel mini trains already in this repo.
That and i also felt adding classic to the name made it a bit unnecessarily lengthy especially if you were to keep it as "Roller Coaster" instead of just "Coaster"
In the end it depends on what you think, but for the sake of consistency i would also suggest updating the classic mini trains as well should you decide to change the names/ids for the classic wooden trains

**Pull request merged**